.github/workflows/_e2e_tests.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           ADMIN_UVICORN_NUM_WORKERS: "2"
           ADMIN_UVICORN_PORT: "8081"
           COMMON_LOG_LEVEL: "DEBUG"
+          COMMON_CONTENT_MAX_SIZE : "10_000_000"
```

Comment:
could you move it to `DATASETS_BASED_CONTENT_MAX_BYTES`? (here, and elsewhere in the code: chart/, datasets_based/, tools/)

- `COMMON_` -> `DATASETS_BASED_`: what we really want would be `WORKER_CONTENT_MAX_SIZE` (or `WORKER_RESPONSE_CONTENT_MAX_SIZE`), but we will create this "namespace" when working on https://github.com/huggingface/datasets-server/issues/737, for now `DATASETS_BASED` is the namespace for all the workers)
- `SIZE` -> `BYTES`: giving the unit avoids errors when modifying the configurations